### ISSUE TITLE
Unexpected semicolon

### ISSUE THREAD

**Issue by tomasbruckner** [airbnb/ts-migrate#62]
Hi, this fails in explicit-any phase

version 0.1.10

simplified version of code. I think props are the problem
```javascript
const InfoPanel = props => {
  const { foo } = props;

  return (
    <div>{foo? 'bar' : 'bar2'}</bar>
  );
};

InfoPanel.propTypes = {
  foo: PropTypes.bool,
};

InfoPanel.defaultProps = {
  foo: false,
};

export default InfoPanel;
```

Stacktrace (from original code)
```
Error: [explicit-any][src\InfoPanel.tsx] Error:
 SyntaxError: Unexpected token, expected ";" (23:22)
    at Object.raise (C:\path\project\node_modules\@babel\parser\lib\index.js:6325:17)
    at Object.unexpected (C:\path\project\node_modules\@babel\parser\lib\index.js:7642:16)
    at Object.semicolon (C:\path\project\node_modules\@babel\parser\lib\index.js:7624:40)
    at Object.parseVarStatement (C:\path\project\node_modules\@babel\parser\lib\index.js:10300:10)
    at Object.parseStatementContent (C:\path\project\node_modules\@babel\parser\lib\index.js:9896:21)
    at Object.parseStatementContent (C:\path\project\node_modules\@babel\parser\lib\index.js:5372:18)
    at Object.parseStatement (C:\path\project\node_modules\@babel\parser\lib\index.js:9829:17)
    at Object.parseBlockOrModuleBlockBody (C:\path\project\node_modules\@babel\parser\lib\index.js:10405:25)
    at Object.parseBlockBody (C:\path\project\node_modules\@babel\parser\lib\index.js:10392:10)
    at Object.parseTopLevel (C:\path\project\node_modules\@babel\parser\lib\index.js:9758:10) {
  pos: 667,
  loc: Position { line: 23, column: 22 }
}
```

**Comment by oleghind** [comment]
Is there any workaround? Happens a lot

**Comment by edsrzf** [comment]
The `react-props` plugin is not dealing well with the arrow function without parentheses. I've created a fix in #81.
